### ISSUE TITLE
IE 11 not Working with Angular MSAL Library

### ISSUE THREAD

**Issue by arunprasathv** [AzureAD/microsoft-authentication-library-for-js#765]
I still see this issue with MSAL Angular library while launching the application in IE11 and console has below Error:

SCRIPT1002: Syntax error
vendor.js (148561,1)

class AuthenticationResult { constructor(token, tokenType) { this._token = ""; this._tokenType = ""; this._token = token; if (tokenType) { this._tokenType = tokenType; } } get token() { return this._token; } set token(value) { this._token = value; } get tokenType() { return this._tokenType; } set tokenType(value) { this._tokenType = value; } }

isn't the classes supposed to be transpiled to es5?

**Comment by sgentile** [comment]
I'm getting the same error, works fine in Chrome, but IE11 broken (this is Azure and it's broken for IE ?)

**Comment by arunprasathv** [comment]
Yes, i know this and i made it to work by recompiling to target es5 and it works in IE11 but the login popup goes in loop. 

Refer :  
![image](https://user-images.githubusercontent.com/7142390/59693525-14779480-91b5-11e9-8b66-19c98c061eb5.png)

**Comment by DarylThayil** [comment]
@jasonnutter  when you get a chance can you look at this and see if we are not compiling down to the right target for ie11?

**Comment by jasonnutter** [comment]
I was able to reproduce this with the MSAL Angular sample, and the cause looks to be that the main export for `msal-angular` isn't the CommonJS version of the library. Fix coming shortly.

**Comment by jasonnutter** [comment]
Fix merged, I'll post here when it has been published. Thanks!

**Comment by arunprasathv** [comment]
@jasonnutter Thanks. This is great!!   #414 closed?

**Comment by jasonnutter** [comment]
Version `0.1.3` of `@azure/msal-angular` has been published to npm with a fix for incompatibility with IE11.

If you have other issues with IE11 that do not get fixed with this version, please file a new issue, thanks!

**Comment by dotnetcanuck** [comment]
@jasonnutter Version 0.1.3, according to the npm registry page, has been deprecated...

```
msal-angular 0.1.3 has been deprecated due to incompatibility with aot compiler. Please revert to 0.1.2.
```

Does `0.1.4-beta.1` include this fix? And is there a timeline on when that version might come out of beta?

**Comment by jasonnutter** [comment]
@dotnetcanuck Yes, `0.1.4-beta.1` has a fix for IE11 compatibility. Please try that version and let us know if it fixes issues you are seeing with IE11. We will promote it to `0.1.4` soon, just waiting to hear back from a few customers confirming that it fixes their issues and doesn't introduce any regressions.

**Comment by aleksandar-kandzhichki** [comment]
@jasonnutter  The `0.1.4-beta.1` seems to fix the issues for me. However logging in with popup does not work. After authentication the popup does not close, the redirectURL is opened in the popup instead of the origin window.
On opening the popup, the origin throws some errors - 
![image](https://user-images.githubusercontent.com/13217235/63265890-4a112a80-c297-11e9-83dd-13ede92f64a4.png)

I suppose this has something to do with maybe polyfills, just unsure what exactly is the issue yet

**Comment by jasonnutter** [comment]
@aleksandar-kandzhichki Yeah that is a known issue we are looking into. For now, our guidance is to use the redirect flow for IE11, instead of popups.

**Comment by mronus** [comment]
@jasonnutter I upgraded the library to 0.1.4-beta.2 but it seems not working. I still have the syntax exception in my console and it prevents everything to be loaded in Angular.

The exception:
![image](https://user-images.githubusercontent.com/2014677/63940824-f19a1400-ca72-11e9-8ef5-3b0b2b2a39a6.png)

The code line where pointed out by the exception:
![image](https://user-images.githubusercontent.com/2014677/63940930-30c86500-ca73-11e9-9136-c935359c19ee.png)

I've also tried different workarounds which are given in github issues related to that problem but no one of them worked.

Any help or hint is highly appreciated :)
